File: trunk/Assets/Levels/Island/Scripts/waterCollision.cs
using UnityEngine;
using System.Collections;

public class waterCollision : MonoBehaviour {
	
	public GameObject [] allPlayers;
	// Use this for initialization
	void Start () {
	
	}
	
	// Update is called once per frame
	void Update () {
	
	}
	void OnTriggerEnter (Collider other)
	{
		StartLap script = other.GetComponent<StartLap>();
		if(script != null)
		{
			script.SlowerCaracter();
		}
		
		//thirdPlayerScript script2 = other.GetComponent<thirdPlayerScript>();
		//if(script2 != null)
		//{
			//script2.SlowerCaracter();
		//}
		
	}
	
	void OnTriggerExit (Collider other)
	{
		StartLap script = other.GetComponent<StartLap>();
		if(script != null)
		{
			script.AccelCaracter();
		} 
		
		//~ thirdPlayerScript script2 = other.GetComponent<thirdPlayerScript>();
		//~ if(script2 != null)
		//~ {
			//~ script2.AccelCaracter();
		//~ }
		
	}
}

File: tags/CopallCart.v6/Assets/Levels/Island/Scripts/waterCollision.js

	// Use this for initialization
	function Start () {
	
	}
	
	// Update is called once per frame
	function Update () {
	
	}
	function OnTriggerEnter (other : Collider)
	{
		var script : StartLap = other.GetComponent(StartLap);
		if(script != null)
		{
			script.SlowerCaracter();
		}
		
		var script2 : Player = other.GetComponent(Player);
		if(script2 != null)
		{
			script2.SlowerCharacter();
		}
		
	}
	
	function OnTriggerExit (other : Collider)
	{
		var script : StartLap = other.GetComponent(StartLap);
		if(script != null)
		{
			script.AcclerateCharacter();
		}
		
		var script2 : Player = other.GetComponent(Player);
		if(script2 != null)
		{
			script2.AcclerateCharacter();
		}
		 
	}
